### PR TITLE
OPS: Upgrade to RN 63

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/8xfb4ytff6kf0hfpcuxdhnyjaw

[download apk](http://filestorage.bluewallet.io:1488/1594258873.apk)

**Comment by marcosrdz** [comment]
A place to test BlueWallet with RN 63. Wouldn't recommend merging into master at least until 63.2. We should keep it up to date with master to avoid big conflicts

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/21k8pk52tjg2jx3fgeq364ybd4

[download apk](http://filestorage.bluewallet.io:1488/1594260868.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6eu1kdg6a9yy0fkuw8jdhcwdh4

[download apk](http://filestorage.bluewallet.io:1488/1594858773.apk)

**Comment by marcosrdz** [comment]
Updated to 0.63.1

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xb1gn9hy8pbaxepc7g7fmg880r

[download apk](http://filestorage.bluewallet.io:1488/1594958282.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/326xhkw3153m091ax1cw6hcj7r

[download apk](http://filestorage.bluewallet.io:1488/1595120338.apk)

**Comment by marcosrdz** [comment]
@Overtorment  @ncoelho  havent found any issues. what do you say?

File: index.js [hunk 1]
```diff
@@ -1,9 +1,7 @@
 import React, { useEffect } from 'react';
 import './shim.js';
-import { AppRegistry, YellowBox } from 'react-native';
+import { AppRegistry } from 'react-native';
 import App from './App';
-YellowBox.ignoreWarnings(['Require cycle', 'Non-serializable values were', "Can't perform a React state update", '{"code":404']);
```
Comment (Overtorment) [inline index.js:5]: yellowbox was removed from rn63 or smth?

Comment (marcosrdz) [reply index.js:5]: Deprecated. Logbox is now king.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xp0hpfabjtfh7hpt71c93mf62c

[download apk](http://filestorage.bluewallet.io:1488/1595256787.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/3ta8g49bx7zq7j32bveq9va8wm

[download apk](http://filestorage.bluewallet.io:1488/1595271990.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/a439m6aju1t3nwfm0xn19e68zr

[download apk](http://filestorage.bluewallet.io:1488/1595284604.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/99r0tmqa5b7ph5u66fawbkk7p8

[download apk](http://filestorage.bluewallet.io:1488/1595559237.apk)

**Comment by Overtorment** [comment]
tests still fail

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/mjqx5wp0t1mdken59eph3yytcc

[download apk](http://filestorage.bluewallet.io:1488/1595628708.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/1ckyw1tfm2m7eemr58hy6ncum4

[download apk](http://filestorage.bluewallet.io:1488/1595632073.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ueju7c1r77mumjwurnfnecjcwc

[download apk](http://filestorage.bluewallet.io:1488/1595711115.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dwrwctp8r9fz112ay13ua356nm

[download apk](http://filestorage.bluewallet.io:1488/1596153717.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4h5v6etebprbx7mgt4u024t3em

[download apk](http://filestorage.bluewallet.io:1488/1596412680.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/9cg47p8ctk5pcxj2tub59e0wyg

[download apk](http://filestorage.bluewallet.io:1488/1596414763.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/b9pndvhx7jqtwxd43ymu1m4tjc

[download apk](http://filestorage.bluewallet.io:1488/1596415843.apk)

**Comment by marcosrdz** [comment]
> tests still fail

Fixed

**Comment by Overtorment** [comment]
loading indicator is invisible across the app (noticed on selftest and wallet creation) :+1: 
image

tested on appetize

File: android/app/build.gradle [hunk 2]
```diff
@@ -156,13 +160,6 @@ android {
         }
     }
 
-    packagingOptions {
-        pickFirst "lib/armeabi-v7a/libc++_shared.so"
```
Comment (Overtorment) [inline android/app/build.gradle:160]: you sure we dont need that?

Comment (marcosrdz) [reply android/app/build.gradle:160]: Yup. Removed from RN's base files.

**Comment by marcosrdz** [comment]
> loading indicator is invisible across the app (noticed on selftest and wallet creation) 👍
> image
> 
> tested on appetize

Apparently its a known android-only bug
https://github.com/facebook/react-native/issues/29378

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/uba16y05uzyevxvmywekeb0jmg

[download apk](http://filestorage.bluewallet.io:1488/1596471256.apk)

**Comment by Overtorment** [comment]
so what do we do? wait till its fixed in RN?

**Comment by ncoelho** [comment]
Give it a bit of time

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/r30an5p73ennzmdt1vq57e7ntw

[download apk](http://filestorage.bluewallet.io:1488/1597108976.apk)

File: android/gradlew.bat [hunk 3]
```diff
@@ -5,7 +5,7 @@
 @rem you may not use this file except in compliance with the License.
 @rem You may obtain a copy of the License at
 @rem
-@rem      https://www.apache.org/licenses/LICENSE-2.0
+@rem      http://www.apache.org/licenses/LICENSE-2.0
```
Comment (Overtorment) [inline android/gradlew.bat]: dafuq

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/bd2prax318whq54bzryju4mte0

[download apk](http://filestorage.bluewallet.io:1488/1600827692.apk)

**Comment by Overtorment** [comment]
e2e fails

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/p88kp226h31x4ybqjrdj9cxv6m

[download apk](http://filestorage.bluewallet.io:1488/1601491876.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/qxmjtnbz965rtrp4k0q39r0cqg

[download apk](http://filestorage.bluewallet.io:1488/1601497015.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/w8auyd8b1e5fgex9an53ttg07m

[download apk](http://filestorage.bluewallet.io:1488/1601498285.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/95k29nmh7xmttm04ntm7m1zd08

[download apk](http://filestorage.bluewallet.io:1488/1601504171.apk)

**Comment by marcosrdz** [comment]
@ncoelho @Overtorment Ready when you are

**Comment by ncoelho** [comment]
List not updating

Screenshot 2020-10-05 at 19 17 01

**Comment by ncoelho** [comment]
Fixed with update

> List not updating
> 
> Screenshot 2020-10-05 at 19 17 01

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ezuhevkdnm9b3mgy7wqeb1j4f8

[download apk](http://filestorage.bluewallet.io:1488/1601919308.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/0gkh8yh5r1jmhjx3uu4mze7fb8

[download apk](http://filestorage.bluewallet.io:1488/1601936957.apk)

**Review by Overtorment — APPROVED** [review]
lgtm

**Pull request merged**